sql/src/main/java/org/apache/druid/sql/calcite/planner/IngestHandler.java

```diff
@@ -350,6 +352,7 @@ public String statementKind()
       return DruidSqlReplace.OPERATOR.getName();
     }
 
+    @Nullable
```

Comment:
I dont think this needs to be on this particular implementation since it is not nullable. Or does it complain if it doesnt match the interface annotations?